lib/screens/edit-profile.dart

```diff
@@ -24,6 +25,14 @@ class EditProfile extends StatefulWidget {
 
 final TextEditingController phnctlr = new TextEditingController();
 final TextEditingController emailctlr = new TextEditingController();
+ScrollController scrollController = ScrollController();
+ScrollController horizontalController = ScrollController();
+int childCount = 10;
+
+int verticalPreviousIndex;
+int horizontalPreviousIndex;
+double verticalChildExtent = 300.0;
+double horizontalChildExtent = 100.0;
```

Comment:
Why are we initializing them outside the class??

Reply:
Didnt notice this, I've move into the class. Thanks